src/lib/recoil/jobs.js

```diff
@@ -0,0 +1,90 @@
+import {atom, noWait, selector, selectorFamily} from "recoil";
+import {httpState, optionsState} from "./base";
+import {careerState, careersParamsState} from "./career";
+
+export const careerJobsState = atom({key: "career-modal-jobs", default: {fetching: false, jobs: []}});
+export const jobOptionsState = atom({key: "job-options", default: {}});
+
+const jobsPathState = selector({
+  get: ({get}) => {
+    const career = get(careerState);
+    const options = get(optionsState).career || {};
+
+    if(options.jobsPath) { return `${options.jobsPath}/${career.id}/jobs`; }
+
+    return `/api/user/careers/${career.id}/jobs`;
+  },
+  key: "jobs-path"
+});
+
+export const jobsQuery = selector({
+  get: async({get}) => {
+    const career = get(careerState);
+    if(!career) { return {}; }
+
+    const http = get(httpState);
+    const path = get(jobsPathState);
+    const careerParams = get(careersParamsState);
+
+    const params = {};
+    careerParams.location && (params.location = careerParams.location);
```

Comment:
A little more readable to just use an `if`

Reply:
Ok updated!